sample-kotlin/build.gradle

```diff
@@ -35,7 +35,9 @@ dependencies {
     compile (project(':library')) {
         exclude module: "support-v13"
     }
-    compile project(':processor')
+
+    provided project(':processor')
 
     kapt project(':processor')
```

Comment:
I'm still confused why there are two dependencies on the `processor` module here. What's the motivation behind including the annotation processor on the compile classpath? Shouldn't `kapt` be enough on its own?